src/gui/LfoControllerDialog.cpp

```diff
@@ -66,48 +66,25 @@ LfoControllerDialog::LfoControllerDialog( Controller * _model, QWidget * _parent
 	setWindowIcon( embed::getIconPixmap( "controller" ) );
 	setFixedSize( 240, 58 );
 	
-	ToolTip::add( this, tr( "LFO Controller" ) );
-
 	m_baseKnob = new Knob( knobBright_26, this );
 	m_baseKnob->setLabel( tr( "BASE" ) );
 	m_baseKnob->move( CD_LFO_BASE_CD_KNOB_X, CD_LFO_CD_KNOB_Y );
-	m_baseKnob->setHintText( tr( "Base amount:" ), "" );
-	m_baseKnob->setWhatsThis( tr("todo") );
-
+	m_baseKnob->setHintText( tr( "Base:" ), "" );
 
 	m_speedKnob = new TempoSyncKnob( knobBright_26, this );
 	m_speedKnob->setLabel( tr( "SPD" ) );
 	m_speedKnob->move( CD_LFO_SPEED_CD_KNOB_X, CD_LFO_CD_KNOB_Y );
-	m_speedKnob->setHintText( tr( "LFO-speed:" ), "" );
-	m_speedKnob->setWhatsThis(
-		tr( "Use this knob for setting speed of the LFO. The "
-			"bigger this value the faster the LFO oscillates and "
-			"the faster the effect." ) );
-
+	m_speedKnob->setHintText( tr( "Modulation speed:" ), "" );
```

Comment:
I understand trying to preserve some uniformity between "Modulation speed" and "modulation amount", but I think this is better as "LFO speed" or "LFO frequency".

It's sort of a two-step thing: you have an LFO, and you adjust _its_ frequency (i.e. the "LFO frequency" - not "Modulation frequency") and then you modulate some other signal with it ("Modulation amount").